readme.md

```diff
@@ -0,0 +1,105 @@
+# Intro
+
+This is MVP for one of our client (UTSEUS) which helps teams on hackathons exchange skills among team members and pay for their help.  It has three modules: web client (React), server (NodeJS) and storage with some business logic on blockchain (ethereum). This doc descriebs how to setup environment and run server backed by blockchian. 
```

Comment:
you misspelled both describes and blockchain...

Reply:
Done